image-search/Models/AlbumGenerator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AlbumGenerator.swift
+//  image-search
+//
+//  Created by Renato Vieira on 08/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+func requestAlbum(url: String, view: ViewController) {
+    if let fakeUrl = URL(string: "http://fakerapiexample.herokuapp.com/album") {
+        URLSession.shared.dataTask(with: fakeUrl) { (data, response, error) in
+            guard let data = data else {
```

Comment:
If you are not using this response I guess you can remove it (like -> { (data, _ , error) in).
Try to do it (I'm not sure if it works). hauha

Reply:
I tried, but gave error!
I left it like this: { (data, _ , error) in and it worked.